metered/src/common/throughput/tx_per_sec.rs

```diff
@@ -1,9 +1,11 @@
 use super::RecordThroughput;
 use crate::hdr_histogram::HdrHistogram;
-use crate::time_source::Instant;
+use crate::clear::Clear;
+use crate::time_source::{Instant, StdInstant};
 use serde::{Serialize, Serializer};
 
-pub struct TxPerSec<T: Instant> {
+/// Non-thread safe implementation of `RecordThroughput`. Use as `RefCell<TxPerSec<T>>`.
```

Comment:
Type at `Uses`

Reply:
I am unclear what you are suggesting here.  Do you want me to change "Use as `RefCell<TxPerSec<T>>`" to "Uses `RefCell<TxPerSec<T>>`"?

Reply:
I was suggesting it matches the doc-comment style used for `AtomicTxPerSec` -- either form seems fine to me!

Reply:
I see.  Alas, they are not the same.  `AtomicTxPerSec` _uses_ a `Mutex` internally.  The comment there merely informs the user of an implementation details.  Whereas `TxPerSec` doesn't use `RefCell` internally.  Rather than comment is informing the user that him, the user, should _use_ `TxPerSec` by wrapping it in a `RefCell`.

I can change the wording to something else if you want.

Reply:
OK, that's fair.  I guess just fixing the typo would be enough then :)